test/edu/csus/ecs/pc2/core/execute/ExecutableTest.java

```diff
@@ -1196,7 +1196,43 @@ public void testSubstituteFiles() throws Exception {
         
         String actual = executable.substituteAllStrings(run, "foo {files}");
         
-        assertEquals("Expected substitution", actual, "foo ISumit.java");
+        assertEquals("Expected substitution", "foo ISumit.java", actual);
+    }
+    
+    /**
+     * test {:ifsuff=xxx} substitution.
```

Comment:
The above line should be updated to reflect the new name of the substitution variable (`{:ensuresuffix=xxx}`)

Reply:
Fixed.